### PR TITLE
sdk/src/camera_itof: don't shift ab data for mp if bpp is 16

### PR DESCRIPTION
**Pull request by SeptimiuVana**
Signed-off-by: SeptimiuVana <Vana.Septimiu@analog.com>

### CODE REVIEW

**Comment by SeptimiuVana** [comment]
> For https://github.com/analogdevicesinc/ToF/blob/add_16bit_mp_ab/sdk/src/cameras/itof-camera/camera_itof.cpp#L777
> 
> Should this be
> 
> uint8_t bitsToShift = 16 - **m_abEnabled**;
> 
> or
> 
> uint8_t bitsToShift = 16 - **m_abBitsPerPixel** ;

It should be uint8_t bitsToShift = 16 - **m_abBitsPerPixel** ; Thank you!

**Comment by SeptimiuVana** [comment]
> For https://github.com/analogdevicesinc/ToF/blob/add_16bit_mp_ab/sdk/src/cameras/itof-camera/camera_itof.cpp#L778
> 
> If **bitsToShift** is 0, it may be useful to skip this loop.

We are doing a check here: https://github.com/analogdevicesinc/ToF/blob/add_16bit_mp_ab/sdk/src/cameras/itof-camera/camera_itof.cpp#L769
That block of code won't be executed if we have 16 bit AB.

File: sdk/src/connections/target/adsd3500_sensor.h [hunk 1]
```diff
@@ -187,8 +187,8 @@ class Adsd3500Sensor : public aditof::DepthSensorInterface,
              {"depth", 1024, 1024},
              {"conf", 1024, 1024},
              {"embedded_header", 1, 128}},
-            1024,
-            4096,
+            2048,
+            2560,
```
Comment (dNechita) [inline sdk/src/connections/target/adsd3500_sensor.h:191]: These are just temporary, right?

Comment (SeptimiuVana) [reply sdk/src/connections/target/adsd3500_sensor.h:191]: We need to keep both resolutions. For now we'll use them like this.

**Review by dNechita — APPROVED** [review]

**Pull request merged**